conf/igenomes.config

```diff
@@ -285,6 +285,7 @@ params {
             bowtie2     = "${params.igenomes_base}/Homo_sapiens/UCSC/hg38/Sequence/Bowtie2Index/"
             star        = "${params.igenomes_base}/Homo_sapiens/UCSC/hg38/Sequence/STARIndex/"
             bismark     = "${params.igenomes_base}/Homo_sapiens/UCSC/hg38/Sequence/BismarkIndex/"
+            hisat2      = "s3://genome-idx/hisat/hg38_genome.tar.gz"
```

Comment:
??

Reply:
Ah, it's their official indexes https://daehwankimlab.github.io/hisat2/download/#h-sapiens